addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -345,7 +353,10 @@ export class SpreadsheetPivotModel extends PivotModel {
         if (groupFieldString.startsWith("#")) {
             const { field } = this.parseGroupField(groupFieldString);
             const { cols, rows } = this._getColsRowsValuesFromDomain(domain);
-            return this._isCol(field) ? cols[cols.length - 1] : rows[rows.length - 1];
+            const fieldValue = this._isCol(field) ? cols[cols.length - 1] : rows[rows.length - 1];
```

Comment:
nitpicking: now that we can use `.at(-1)` I would use that rather than `[cols.length - 1]`. Shorter and more readable IMO